src/config/config.js

```diff
@@ -4,6 +4,11 @@ const config = {
   dayLabels: [
     'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday',
   ],
+  postTable: {
+    linkPrefix: 'https://reddit.com/',
```

Comment:
Currently you have two `/` after reddit.com in the links. `https://www.reddit.com//r/javascript/comments/elt6gd/were_killing_the_mobile_web/`
Somehow reddit works a bit like this, but not completely :smile: